src/styles/styles.js

```diff
@@ -3720,6 +3720,10 @@ const styles = (theme) => ({
     receiptDropHeaderGap: {
         backgroundColor: theme.receiptDropUIBG,
     },
+
+    checkBoxWithLabelCheckBoxStyle: {
```

Comment:
The `checkbox` name for the component is not camel case.

```suggestion
    checkboxWithLabelCheckboxStyle: {
```